toolchains/iOS.framework.cmake

```diff
@@ -3,7 +3,7 @@ include(${CMAKE_SOURCE_DIR}/toolchains/iOS.toolchain.cmake)
 message(STATUS "Build for iOS archs " ${CMAKE_OSX_ARCHITECTURES})
 
 set(FRAMEWORK_NAME TangramMap)
-set(FRAMEWORK_VERSION "1.0")
```

Comment:
So, just to follow up - this is going to be updated to use `-dev` when we're in development, and `-alpha`/`-beta` tags to line up with appropriate level of release status.